_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
+> ## Content of subfolders
+>
+> * ``plots``: the location for all plots, split by individual diagnostics and fields.
+> * ``preproc``: this folder contains all the preprocessed data and metadata.yml interface files. Note that by default this directory will be deleted after each run because most users will only need the results from the diagnostic scripts.
+> * ``run``: this folder includes all log files, a copy of the recipe, a summary of the resource usage, and the settings.yml interface files, resource_usage.txt and temporary files created by the diagnostic scripts.
+> * ``work``: a place for any diagnostic script results that are not plots, e.g. files in NetCDF format (depends on the diagnostic script).
+>
+> We explain more about output in the next [lesson](https://esmvalgroup.github.io/tutorial/04-toy-example/index.html)
+{: .callout}
 
-.. note::
+## Auxiliary data directory
 
-   The ``config-user.yml`` file is specified as argument at run time, so it is
-   possible to have several available with different purposes: one for
-   formalised runs, one for debugging, etc...
+The ``auxiliary_data_dir`` setting is the path to place any required additional auxiliary data files. This location allows us to tell the diagnostic script where to find the files if they can not be downloaded at runtime. This option is not for model or observational datasets, rather it is for data files used in plotting such as coastline descriptions and so on.
 
+~~~YAML
+auxiliary_data_dir: ~/auxiliary_data
+~~~
 
-{% include links.md %}
+## Output settings
 
+These settings are used to inform ESMValTool about your preference. You can turn on or off the setting by ``true`` or ``false`` values. Most of these settings are fairly self-explanatory, ie:
+
+~~~YAML
+# Diagnostics create plots? [true]/false
+write_plots: true
+# Diagnositcs write NetCDF files? [true]/false
+write_netcdf: true
+# Set the console log level debug, [info], warning, error
+log_level: info
+# Exit on warning (only for NCL diagnostic scripts)? true/[false]
+exit_on_warning: false
+# Plot file format? [png]/pdf/ps/eps/epsi
+output_file_type: png
+# Use netCDF compression true/[false]
+compress_netcdf: false
+# Save intermediary cubes in the preprocessor true/[false]
+save_intermediary_cubes: false
+# Remove the preproc dir if all fine
+remove_preproc_dir: true
+# Path to custom config-developer file, to customise project configurations.
+# See config-developer.yml for an example. Set to [null] to use the default
+# config_developer_file: null
+# Get profiling information for diagnostics
+# Only available for Python diagnostics
+profile_diagnostic: false
+~~~
+
+> ## Make your own configuration file
+>
+> It is possible to have several configuration files with different purposes, for example: config-user_formalised_runs.yml, config-user_debugging.yml
+{: .callout}
+
+> ## Different settings
+>
+> In the configuration file, which settings are useful to store preprocessed data?
```

Comment:
I would probably rephrase to "In the configuration file, which settings are usefule to make sure preprocessed data is stored when ESMValTool is run?